module/VuFind/src/VuFind/Cover/RouterFactory.php

```diff
@@ -77,6 +77,7 @@ public function __invoke(
             $base = ($container->get('ViewRenderer')->plugin('url'))('cover-show');
         }
         $coverLoader = $container->get(\VuFind\Cover\Loader::class);
-        return new $requestedName($base, $coverLoader);
+        $config = $container->get(\VuFind\Config\PluginManager::class)->get('config')->toArray()['Content'];
```

Comment:
Probably worth adding a default fallback for safety (though this section is unlikely to be missing in practice):

```suggestion
        $config = $container->get(\VuFind\Config\PluginManager::class)->get('config')->toArray()['Content'] ?? [];
```